docs/readme.md

```diff
@@ -13,6 +13,7 @@ const chromeLauncher = require('lighthouse/chrome-launcher');
 
 function launchChromeAndRunLighthouse(url, flags = {}, config = null) {
   return chromeLauncher.launch().then(chrome => {
+    flags.host = 'localhost';
```

Comment:
I think we should actually remove this, sorry :) We don't document any of the other possible flags here, and we want to keep this example as simple as possible for copy/pasting to get started.

We should start a separate section to document the flags (or at least point to the CLI flags as the same as what this accepts), but that's a change for a different PR.